### ISSUE TITLE
Final Project Feedback

### ISSUE THREAD

**Issue by kayroy247** [mkeaglejr/blockchain-developer-bootcamp-final-project#1]
Hey @mkeaglejr, congratulations on finishing your final project. I am a mentor who is assisting with the grading.

We'll be approving it on the course itself in a week or two, but here's your feedback in the meanwhile:

#### Frontend Comments

- Good job. However, it would have been better if the Dapp had a connect button for users to connect because the page only tries to connect to the user wallet on page load. 
- The front end is supposed to also show a supported network indicator for user experience. 
- After registering the user, the page has to be reloaded before the form appears. 
- The frontend is not updated with any data from the contract
-  And also there is no update on the frontend after the completion of the transaction

#### Test comments
- Well-structured tests with description, comments, and beforeEach hook.
- The test ran successfully and all 5 tests passed.

**Comment by Project-MKEAGLE** [comment]
Thank you for the feedback. Am I still able to make these changes before
the final grading?

**Comment by kayroy247** [comment]
@mkeaglejr Yes, you can make the changes before the final grading. Please reach out to coogan.brennan@consensys.net when you have implemented the changes.
